web/src/app/models/user.model.ts

```diff
@@ -0,0 +1,23 @@
+export interface User {
+  displayName: string;
+  photoUrl: string;
+  uid: string;
+  creationAt: string;
+  lastSignInAt: string;
+  credentials: {
+    accessToken: string,
+    providerId: string,
+    refreshToken: string,
+    secret?: string,  // Optional field
+  };
+  additionalUserInfo: {
+    profile: {
+      createdAt: string,
+      updatedAt: string,
+      followers: 0,
+      following: 0,
```

Comment:
I think this should have a type then a default value...

```typescript
following: number = 0
```

Reply:
Hi Eddie, 
This ts files is an interface. The above mentioned property can be used only if it is a class. Do you want me to change it to class ?
Another way is we can use a class to extend the interface and override the properties there.

Reply:
Oh you are right, keep it as an interface 👍